test/esp/resources/custom_signature_test.rb

```diff
@@ -205,7 +205,7 @@ class CustomSignatureTest < ActiveSupport::TestCase
           end
 
           should 'return errors' do
-            @custom_signature.run(external_account_id: 999)
+            @custom_signature.run(external_account_id: 99999999999)
```

Comment:
Style/NumericLiterals: Separate every 3 digits in the integer portion of a number with underscores(_).